doc/man3/EVP_DigestVerifyInit.pod

```diff
@@ -9,9 +9,10 @@ EVP_DigestVerifyFinal, EVP_DigestVerify - EVP signature verification functions
 
  #include <openssl/evp.h>
 
- int EVP_DigestVerifyInit_ex(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
-                             const char *mdname, const char *props,
-                             EVP_PKEY *pkey, EVP_SIGNATURE *signature);
+ int EVP_DigestVerifyInit_ex(OPENSSL_CTX *libctx, EVP_MD_CTX *ctx,
```

Comment:
these have to fixed too right?

Reply:
Oops. Fixed. Also updated the rest of the params because they were out of date and adjusted the text to match.